deploy/lib/querier.libsonnet

```diff
@@ -0,0 +1,89 @@
+local kube = import 'kube-libsonnet/kube.libsonnet';
+
+{
+    querier_deployment:
+        local name = $._config.querier.name;
+        local labels = $._config.querier.labels;
+
+        # Arguments
+        local extraArgs = $._config.querier.extraArgs;
+        local consul_uri = $._config.consul.name + '.' + $._config.namespace + '.svc.cluster.local:8500';
+        local args = [
+            '-server.http-listen-port=80',
+            '-querier.frontend-address=query-frontend.' + $._config.namespace + '.svc.cluster.local:9095',
+            '-querier.batch-iterators=true',
+            '-querier.ingester-streaming=true',
+            '-config-yaml=/etc/cortex/schemaConfig.yaml',
+            '-consul.hostname=' + consul_uri,
+        ];
+        
+        # Environment Variables
+        local env = $._config.ingester.env;
+        local envKVMixin = $._config.querier.envKVMixin;
+        local extraEnv = [{name: key, value: extraEnv[key]} for key in std.objectFields(envKVMixin)];
```

Comment:
should be: `local extraEnv = [{name: key, value: envKVMixin[key]} for key in std.objectFields(envKVMixin)];`